doc/whats_new/v1.3.rst

```diff
@@ -141,6 +141,15 @@ Changelog
 :mod:`sklearn.preprocessing`
 ............................
 
+- |Enhancement| Adds a `class_names` parameter to
+  :func:`tree.export_text`. This allows specifying the parameter `class_names`
+  for each target class in ascending numerical order.
+  :pr:`25387` by :user:`William M <Akbeeh>`, :user:`Guillaume Lemaitre <glemaitre>`, and
+  :user:`crispinlogan <crispinlogan>`.
+
+:mod:`sklearn.tree`
```

Comment:
I think that you mixed the entry here. Could you make sure to have `sklearn.tree` in the right alphabetic order and your entry just below it?